src/utils/html/Container.ts

```diff
@@ -57,15 +57,11 @@ export default class Container {
             container.appendChild(canvas);
             const loading = document.getElementById("loading");
             document.body.insertBefore(container, loading);
-            var containerEvent = new Event("containerEvent");
```

Comment:
Good idea!In this way we reduce the code size, but maybe another time please make a separate PR. :slightly_smiling_face: